susemanager-utils/susemanager-sls/salt/channels/init.sls

```diff
@@ -18,12 +18,36 @@ mgrchannels_susemanagerplugin_conf:
     - mode: 644
 {%- endif %}
 
+{%- if grains['os_family'] == 'Debian' %}
+mgrchannels_susemanagermethod:
+  file.managed:
+    - name: /usr/lib/apt/methods/susemanager
+    - source:
+      - salt://channels/apt-transport-susemanager/susemanager
+    - user: root
+    - group: root
+    - mode: 755
+    - require:
+      - file: mgrchannels_channeltokens
+
+mgrchannels_channeltokens:
+  file.managed:
+    - name: /etc/susemanager.conf
+    - source:
+      - salt://channels/debian_channel_tokens.repo
+    - user: root
```

Comment:
AFAIK you use jinja templating in this file. So you need to set it in file.managed. Otherwise it wil just 1:1 deploy the file as it is.
```
file.managed:
  - template: jinja
...
```